client/blocks/site/README.md

```diff
@@ -6,7 +6,7 @@ This component displays a Site item using the data structure provided by sites-l
 #### How to use:
 
 ```js
-var Site = require( 'my-sites/site' );
+var Site = require( 'blocks/site' );
```

Comment:
Would rather get this with the minimal possible change to avoid breaking anything. :)